src/org/opendatakit/briefcase/model/ProxyConnection.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2011 University of Washington.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.opendatakit.briefcase.model;
+
+/**
+ * This class models a proxy connection.
+ * 
+ * @author rclakmal@gmail.com
+ *
+ */
+public class ProxyConnection {
+	private String host;
+	private Integer port;
+	private String proxyType;
```

Comment:
I thought about removing the enum too. But then I thought it is much clearer/extendable this way to maintain the proxyType without adding string constants to an external file.